lib/vscode/src/vs/workbench/contrib/webview/browser/pre/service-worker.js

```diff
@@ -256,7 +256,7 @@ async function processResourceRequest(event, requestUrl) {
 
 	const firstHostSegment = requestUrl.hostname.slice(0, requestUrl.hostname.length - (resourceBaseAuthority.length + 1));
 	const scheme = firstHostSegment.split('+', 1)[0];
-	const authority = firstHostSegment.slice(scheme.length + 1); // may be empty
+	const authority = firstHostSegment.slice(scheme.length + 1) + port ? (":" + port) : ""
```

Comment:
Looks like CodeTogether cut out before it got the rest of the changes :stuck_out_tongue: 

Anyway, together with the other change this would look like:

```suggestion
	const authority = firstHostSegment.slice(scheme.length + 1) + requestUrl.port ? (":" + requestUrl.port) : ""; // may be empty
```